index.bs

```diff
@@ -661,14 +667,19 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Assert: <code>|options|.{{CredentialCreationOptions/publicKey}}</code> is [=present=].
 
+1. If <var ignore>OriginWithAncestors</var> is `false`, return a "{{NotAllowedError}}" {{DOMException}}.
```

Comment:
You lost the `same` here in the last commit. :)

Reply:
LOL!!  "haste makes waste..."  thx for catching that   o_O